app/src/components/page/LanguageSwitcher/index.tsx

```diff
@@ -0,0 +1,70 @@
+/** globe icon with a dropdown where users can select language */
+import React, { SetStateAction } from 'react';
+import { Menu, Dropdown, Button } from 'antd';
+import { GlobalOutlined } from '@ant-design/icons';
+import { getConfig, LanguageCode, setConfig } from '@opensrp/pkg-config';
+import { SUPPORTED_LANGUAGES } from '../../../configs/env';
+import i18n from '../../../mls';
+import { MenuClickEventHandler } from 'rc-menu/lib/interface';
+
+/** describes object representation of language options */
+export type LanguageOptions = {
+  [key in LanguageCode]?: string;
+};
+
+/** enum of all possible language options */
+export const languageOptions: LanguageOptions = {
+  en: 'English',
+  fr: 'Français',
```

Comment:
Do these need to be translatable?

Reply:
I don't think so, so that they user can easily recognize(in the language switcher) the language that they would wish the app to be in. I think they should be non-translatable string constants,